packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -840,11 +842,19 @@ const CFDPasswordModal = ({
 
         const category_label = category === 'real' ? localize('real') : localize('demo');
         const type_label =
-            getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][
-                type as keyof TMtCompanies['demo' | 'real']
-            ].short_title;
-        const deriv_x_type_label =
-            getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']].short_title;
+            platform === CFD_PLATFORMS.MT5
+                ? getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][
+                      type as keyof TMtCompanies['demo' | 'real']
+                  ].title
+                : platform === CFD_PLATFORMS.DXTRADE
+                ? getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']]
+                      .short_title
+                : platform === CFD_PLATFORMS.DERIVEZ
+                ? getDerivezCompanies()[category as keyof TDerivezCompanies][
+                      type as keyof TDerivezCompanies['demo' | 'real']
+                  ].short_title
+                : '';
```

Comment:
Can we do like a switch case for this 

```suggestion
           let type_label = '';

switch (platform) {
  case CFD_PLATFORMS.MT5:
    const mtCompanies = getMtCompanies(show_eu_related_content);
    type_label = mtCompanies[category][type].title;
    break;
  case CFD_PLATFORMS.DXTRADE:
    const dxCompanies = getDxCompanies();
    type_label = dxCompanies[category][type].short_title;
    break;
  case CFD_PLATFORMS.DERIVEZ:
    const derivezCompanies = getDerivezCompanies();
    type_label = derivezCompanies[category][type].short_title;
    break;
  default:
    type_label = '';
    break;
}
```